vital_sqi/data/signal_sqi_class.py

```diff
@@ -13,7 +19,116 @@ def __init__(self, wave_type=None, signals=None, sampling_rate=None,
         self.wave_type = wave_type
         self.sqi_indexes = sqi_indexes
         self.info = info
+        self.isSplit = False
+        self.isFiltered = False
+        self.SQIComputed = False
+        self.unfilteredsignal = None
+
+    def split_to_segments(self, minute_remove=0.5, segment_duration=30):
```

Comment:
can you alter the `split_to_segments` in `utils.segment_split.py` (in the current main) to output both files and segment variables? I think, this function is better in common than attached to SignalSQI.